test/core/application/app_config_test.cpp

```diff
@@ -22,6 +49,10 @@ class AppConfigurationTest : public testing::Test {
   }
 
   void SetUp() override {
+    std::ofstream file("./config.json",
```

Comment:
Please, use temporary firectory for that. See [this code](https://github.com/soramitsu/kagome/blob/0b08f2f0b178a386f64aa2095c99e7096ba1f26a/test/integration/block/application_test_suite.hpp#L18) for example.